### PR TITLE
Improve stripe support

### PR DESCRIPTION
**Pull request by makasim**
- authorize
- sync
- capture
- better status handling
- ...

for reference: https://github.com/Payum/Stripe/pull/3

### CODE REVIEW

File: src/Payum/Stripe/Action/AuthorizedAction.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+use Payum\Core\Action\PaymentAwareAction;
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\Request\Authorize;
+use Payum\Core\Request\Sync;
+use Payum\Stripe\Request\Api\CreateCharge;
+use Payum\Stripe\Request\Api\ObtainToken;
+
+class AuthorizeAction extends PaymentAwareAction
```
Comment (maddo) [inline src/Payum/Stripe/Action/AuthorizedAction.php]: There's a typo on this filename!

Comment (maddo) [inline src/Payum/Stripe/Action/AuthorizedAction.php]: `src/Payum/Stripe/Action/AuthorizedAction.php` should be
`src/Payum/Stripe/Action/AuthorizeAction.php` (for clarity)

Comment (makasim) [inline src/Payum/Stripe/Action/AuthorizedAction.php]: fixed.

Comment (maddo) [inline src/Payum/Stripe/Action/AuthorizedAction.php]: :+1:

**Pull request closed without merging**